lib/api/src/lib.rs

```diff
@@ -217,3 +222,23 @@ impl CompiledModule for Module {
         Ok(())
     }
 }
+
+// Below this line is things copied from `wasmer-runtime` to make the C API work.
+// All these additions should be reviewed carefully before shipping.
+
+/// Compile WebAssembly binary code into a [`Module`].
+/// This function is useful if it is necessary to
+/// compile a module before it can be instantiated
+/// (otherwise, the [`instantiate`] function should be used).
+///
+/// [`Module`]: struct.Module.html
+/// [`instantiate`]: fn.instantiate.html
```

Comment:
I thought Rust docs did that automatically?

Reply:
I believe it's a nightly rustdoc thing; docs.rs uses it by default.  But it relies on it being in scope; either way that doc comment was just copied and pasted from runtime

Reply:
It's called intra-doc link resolution. It's a nightly only feature. Also, we may want to add:

```rust
![deny(missing_docs, intra_doc_link_resolution_failure)]
```

everywhere. It protects again broken links.

docs.rs uses nighty to generate the doc, by the way.